plugins/src/messages.rs

```diff
@@ -63,6 +64,7 @@ pub struct GetManifestCall {}
 #[derive(Deserialize, Debug)]
 pub(crate) struct InitCall {
     pub(crate) options: HashMap<String, Value>,
+    pub(crate) configuration: Configuration,
```

Comment:
Would you prefer I just use `Value` here?

Reply:
Maybe an HasMap<String, Value> or keep the Configuration here! But if I'm not wrong @cdecker wants this library strongly typed, perhaps the struct is better?

Reply:
Went with "configuration". I prefer that too.